content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -17,83 +20,148 @@ locations.
+======================
 
-The *Opened* rate measures the percentage of emails opened by recipients against the total number of
-sent emails.
+Individual metrics can be analyzed by creating a report. To begin, click on the smart button of the
+desired metric.
 
-A high open rate may indicate that the subject line was timely, compelling, and successfully
-prompted the recipients to view the message.
+Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of the search bar to see
+the list of filtering and grouping parameters.
 
-A low open rate may indicate the subject line failed to capture the recipients' interest. It could
-also indicate the email ended up in a spam or junk folder due to having a poor sender reputation or
-failing to :doc:`configure the proper DNS records <../../general/email_communication/email_domain>`.
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only the
+results that fit the filter. For example, selecting the :guilabel:`Bounced` filter only shows emails
+that could not be delivered.
 
-Replied rate
-------------
+:guilabel:`Group By`, found in the middle column, is used to organize the results into groups, and
+can be used with or without filters.
 
-The *Replied* rate measures the percentage of recipients who responded to the email directly against
-the total number of sent emails.
+.. note::
+  Setting multiple :guilabel:`Group By` options creates nested groups, according to which option
+  is selected first. For example, selecting :guilabel:`Mailing Lists`, followed by
+  :guilabel:`Bounce`, in the :guilabel:`Group By` column, sorts all results *first* by the mailing
+  list, *then* by whether an email was bounced.
 
-A high replied rate may indicate the email resonated with recipients, prompting them to take action
-or provide feedback.
+  This can be verified by looking at the direction, and order, of the selections in the group tile
+  that appears in the search bar.
 
-A low replied rate may suggest that the message lacked relevance or did not contain a clear call to
-action.
+.. example::
+   A monthly newsletter has been sent out, and 6.9% of the sent emails were bounced.
 
-Clicked rate
-------------
+   .. image:: analyze_metrics/newsletter-metrics.png
+      :align: center
+      :alt: The metrics smart buttons of the newsletter.
 
-The *Clicked* through rate (CTR), measures the percentage of recipients who clicked on a link within
-the email against the total number of sent emails.
+   To see what these bounced recipients have in common, the records are grouped using a custom group
+   targeting :guilabel:`Mailing Lists`, which groups all records by the mailing lists they are on.
+   The records are then filtered using a custom filter with the rule `Created on >= 07/01/2024
+   00:00:00`, when the mailing list was last checked. This filter only includes recipients
+   that have been created on or after July 1st, 2024, in the report.
```

Comment:
I think this example might benefit from some clarification.

When I click on 'Bounced' to follow this example, and I click 'Add Custom Group', it reveals a long list of options, but 'Mailing Lists' isn't one of them. Perhaps that's because, on the runbot I'm using for the review, there is only one contact appearing on the 'Bounced' page, making such a grouping impossible. Either way, I'd suggest confirming the presence of this during your revisions.

And, I'd suggest either ref-linking to a doc about how to create a custom filter, or providing the reader with a brief breakdown of how to create a custom filter, so they can be armed with that knowledge, should they want to create one of their own.
```suggestion
   To see what these bounced recipients have in common, the records are grouped using a custom group
   targeting :guilabel:`Mailing Lists`, which groups all records by the mailing lists they are on.
   The records are then filtered using a custom filter with the rule `Created on >= 07/01/2024
   00:00:00`, when the mailing list was last checked. This filter only includes recipients
   that have been created on, or after, July 1st, 2024, in the report.
```